ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -145,6 +145,40 @@ type structure_ethtool = {
 };
 
 @documentation{
+    interface plugin for vxlan support via initscripts-vxlan
+}
+type structure_interface_plugin_vxlan = {
+    @{VXLAN Network Identifier (or VXLAN Segment ID); derived from devicename vxlan[0-9] if not defined}
+    'vni' ? long(0..) # up to 16M
+    @{multicast ip to join}
+    'group' ? type_ip
+    @{destination IP address to use in outgoing packets}
+    'remote' ? type_ip
+    @{source IP address to use in outgoing packets}
+    'local' ? type_ip
+    @{UDP destination port}
+    'dstport' ? long(0..)
```

Comment:
Hmm... in the general case it should probably be `(2..65535)` to avoid use of magical wild-card ports.

Should this default to either the linux default of `8472` or the IANA assigned `4789`?